dashboard/src/components/AppView/AppView.tsx

```diff
@@ -195,82 +203,96 @@ export default function AppView() {
     resourceRefs;
   const revision = app?.revision ?? 0;
   const icon = appDetails?.iconUrl ?? placeholder;
+
   return (
-    <section>
-      <PageHeader
-        title={releaseName}
-        titleSize="md"
-        helm={true}
-        icon={icon}
-        buttons={[
-          <UpgradeButton
-            key="upgrade-button"
-            cluster={cluster}
-            namespace={namespace}
-            releaseName={releaseName}
-            releaseStatus={app?.status}
-            plugin={pluginObj}
-          />,
-          <RollbackButton
-            key="rollback-button"
-            cluster={cluster}
-            namespace={namespace}
-            releaseName={releaseName}
-            revision={revision}
-            releaseStatus={app?.status}
-            plugin={pluginObj}
-          />,
-          <DeleteButton
-            key="delete-button"
-            cluster={cluster}
-            namespace={namespace}
-            releaseName={releaseName}
-            releaseStatus={app?.status}
-          />,
-        ]}
-      />
-      {error &&
-        (error.constructor === DeleteError ? (
-          <Alert theme="danger">Unable to delete the application. Received: {error.message}</Alert>
-        ) : (
-          <Alert theme="danger">An error occurred: {error.message}</Alert>
-        ))}
-      {!app || !app?.status?.userReason ? (
-        <LoadingWrapper loadingText={`Loading ${releaseName}...`} />
+    <LoadingWrapper loaded={!!app} loadingText="Retrieving application..." className="margin-t-xl">
+      {!app || !app?.installedPackageRef ? (
+        <Alert theme="danger">There is a problem with this package</Alert>
       ) : (
-        <Row>
```

Comment:
This was allowing wrong errors during a normal load. Indeed, in the video I recorded, there's a red message complaining about a plugin not being set or sth like that. Moved to avoid this behavior